index.html

```diff
@@ -41,6 +41,196 @@ <h1 class="title">Hey There. I'm Meshu</h1>
                 <li><a href="#"><img src="image/medium.png" alt="medium"></a></li>
             </ul>
         </section>
+        <section class="work">
+            <header class="work-header">
+                <h1 class="title">My Recent Works</h1>
+                <hr class="line">
+            </header>
+
+            <div class="feature-section">
+                <figure class="feature-image">
+                    <img src="image/featuer.png" alt="featuer image">
+                </figure>
+
+                <h2 class="story-title">Multi-Post Stories</h2>
+                <p class="story-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                    been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                    standard dummy text.
+                </p>
+                <ul class="tools">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>ruby</li>
+                </ul>
+                <button type="button" class="story-button">See Project</button>
+            </div>
```

Comment:
- [ ] Please, ensure that your card button matches the Figma template. This is due to the height you applied to it and the text `font-size` it has.

![image](https://user-images.githubusercontent.com/80612925/177006532-c4df69da-0f26-4806-b38f-af0466939d59.png) ![image](https://user-images.githubusercontent.com/80612925/177006552-60dfd3ab-2af9-4877-956e-9114f3cd11be.png)


- [ ] Also, it would be better to follow the button styling provided in the template. Ensure that they have the `:hover`, `:active`, `:disabled` actions. Check out the design below:

**Figma buttons**
![image](https://user-images.githubusercontent.com/80612925/177006493-1826d986-9530-4467-ad4f-5baf1368333d.png)

Reply:
> * [ ]  Please, ensure that your card button matches the Figma template. This is due to the height you applied to it and the text `font-size` it has.
> 
> ![image](https://user-images.githubusercontent.com/80612925/177006532-c4df69da-0f26-4806-b38f-af0466939d59.png) ![image](https://user-images.githubusercontent.com/80612925/177006552-60dfd3ab-2af9-4877-956e-9114f3cd11be.png)
> 
> * [ ]  Also, it would be better to follow the button styling provided in the template. Ensure that they have the `:hover`, `:active`, `:disabled` actions. Check out the design below:
> 
> **Figma buttons** ![image](https://user-images.githubusercontent.com/80612925/177006493-1826d986-9530-4467-ad4f-5baf1368333d.png)

Thank you for your constrictive comment. But I followed the design. or I didn't understand your suggestion?